src/constants/AGENTS.md

```
@@ -0,0 +1,28 @@
+<!-- Parent: ../AGENTS.md -->
+<!-- Generated: 2026-07-12 | Updated: 2026-07-12 -->
+
+# constants
+
+## Purpose
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)